app/crud.py

```diff
@@ -70,3 +70,14 @@ def delete_transaction(db: Session, transaction_id: int):
         db.delete(db_transaction)
         db.commit()
     return db_transaction
+
+
+def get_revenue_for_period(db: Session, start_date: str, end_date: str) -> float:
```

Comment:
This thing is good. But I think that having things templated would be really better. Means, you can still keep `get_revenue_for_period` but make something more objective like `get_revenue_for_year`, `get_revenue_for_week`, `get_revenue_for_month`. In terms of realization, you can use `get_revenue_for_period` but in endpoint specify the time.